cmd/object-api-datatypes.go

```diff
@@ -79,13 +79,13 @@ const (
 // ObjectsHistogramIntervals is the list of all intervals
 // of object sizes to be included in objects histogram.
 var ObjectsHistogramIntervals = []objectHistogramInterval{
-	{"LESS_THAN_1024_B", 0, humanize.KiByte - 1},
-	{"BETWEEN_1024_B_AND_1_MB", humanize.KiByte, humanize.MiByte - 1},
-	{"BETWEEN_1_MB_AND_10_MB", humanize.MiByte, humanize.MiByte*10 - 1},
-	{"BETWEEN_10_MB_AND_64_MB", humanize.MiByte * 10, humanize.MiByte*64 - 1},
-	{"BETWEEN_64_MB_AND_128_MB", humanize.MiByte * 64, humanize.MiByte*128 - 1},
-	{"BETWEEN_128_MB_AND_512_MB", humanize.MiByte * 128, humanize.MiByte*512 - 1},
-	{"GREATER_THAN_512_MB", humanize.MiByte * 512, math.MaxInt64},
+	{"< 1024B", 0, humanize.KiByte - 1},
+	{"> 1024B", humanize.KiByte, humanize.MiByte - 1},
+	{"> 1MB", humanize.MiByte, humanize.MiByte*10 - 1},
+	{"> 10MB", humanize.MiByte * 10, humanize.MiByte*64 - 1},
+	{"> 64MB", humanize.MiByte * 64, humanize.MiByte*128 - 1},
+	{"> 128MB", humanize.MiByte * 128, humanize.MiByte*512 - 1},
+	{"> 512MB", humanize.MiByte * 512, math.MaxInt64},
```

Comment:
By the way, if we will use '>' style, then we should update calculation algorithm as well.

For example, for '> 1MB', we should consider all files with size bigger than 1Mb, like 500mb, 1gb, etc..

Reply:
Yeah, hence my proposal.